### ISSUE TITLE
the loss tend to be nan

### ISSUE THREAD

**Issue by ray-lee-94** [jiangtaoxie/fast-MPN-COV#4]
when i used mpncovresnet50 and MPNCOV， the train converged.
But if i change the backbone to resnet* or VGG*， keeping the MPNCOV unchanged, the train loss is nan.
Beside, my dataset is for a  mini-FGVC task. It contains 9 classes with extra-unbalanced. When i fine-tuning 
within two stage, The test acc is about 78, which is lower than plain vgg. Could you give me some advice?
Thank you for your amazing work.

**Comment by jiangtaoxie** [comment]
@VCBE123 
`>change the backbone to resnet* or VGG*， keeping the MPNCOV unchanged, the train loss is nan.`

There is a bug when reconstructing resnet*, it's fixed. But for VGG*, I did not observe this problem, can you show me your settings in `train.sh`?

`>test acc is lower than plain vgg`

Can you show me your experimental settings? Maybe need to adjust learning rate or weight decay.
